content/hacking-atom/sections/package-word-count.md

```diff
@@ -133,9 +127,9 @@ An optional `styleSheets` array in your `package.json` can list the style sheets
 
 ##### Keymaps
 
-You can provide key bindings for commonly used actions for your extension, especially if you're also adding a new command. In our new package, we have a keymap filled in for us already in the `keymaps/wordcount.cson` file:
+You can provide key bindings for commonly used actions for your extension, especially if you're also adding a new command. In our new package, we have a keymap filled in for us already in the `keymaps/wordcount.json` file:
 
-```coffee
+```javascript
```

Comment:
This (and all other CSON examples) need to be kept as coffee until it is converted to JSON.